CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 
 ### Fixed
+- Terminate gracefully on SIGTERM - @recmo
+- Use exec in Dockerfile - @recmo
```

Comment:
At least until now, the changelog contained info strictly about the executable, docker file is more like documentation/example (that should be moved somewhere else, in time). What i am trying to say is the line about docker should not be here :)